client/scss/components/_transporter.scss

```diff
@@ -0,0 +1,84 @@
+.transporter {
+  background: color('white');
+  border-radius: $border-radius-unit;
+  padding: $vertical-space-unit map-get($gutter-width, 'small');
+  margin-bottom: $vertical-space-unit;
+
+  @include respond-to('medium') {
+    padding: $vertical-space-unit map-get($gutter-width, 'medium');
+  }
+
+  @include respond-to('large') {
+    padding: $vertical-space-unit map-get($gutter-width, 'large');
+  }
+}
+
+.transporter__item:first-child:last-child {
+  @include respond-to('large') {
+    max-width: 50%;
+  }
+}
+
+// font-sizes when there are 4 items or fewer
+.transporter__item:nth-last-child(-n+4):first-child,
+.transporter__item:nth-last-child(-n+4):first-child ~ .transporter__item {
+  .promo__type {
+    @include respond-to('large') {
+      @include font('HNL5');
+    }
+  }
+
+  .promo__title {
+    @include respond-to('large') {
+      @include font('WB6');
+    }
+  }
+}
+
```

Comment:
Not sure about this bit. You can see from the screenshots that the font sizes change depending on how many items are in the transporter.

This takes care of that, but means mixing the .promo styles in this .transporter file.

What do people think?